linku_frontend/linku/__test__/Appliers.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
I think that the test file should be written to understand the role of the component.